packages/components/Avatar/src/useAvatar.ts

```diff
@@ -118,6 +118,7 @@ export const getInitials = (name: string): string => {
   );
   const words = removeRedundantCharacters(name);
   const wordsLength = words.length;
+  if (!wordsLength) return '';
```

Comment:
styling nit: can we put newlines around this so that it's easier to see apart from all the const declarations?

Reply:
Also a personal preference is to have the bodies of conditional statements wrapped in brackets to make them easier to see, but it's up to you if you want to add them.

Reply:
Yeah, I think with brackets it'll be better